prediction_market_agent/data_models/market_data_models.py

```diff
@@ -131,7 +131,7 @@ class ManifoldMarket(BaseModel):
     creatorUsername: str
     isResolved: bool
     lastBetTime: datetime
-    lastCommentTime: datetime
+    # lastCommentTime: datetime # Not always present
```

Comment:
Then it could be optional?

```suggestion
    lastCommentTime:t.Optional[datetime]
```